src/app/content/pages/components/add-products/add-products.component.html

```diff
@@ -0,0 +1,101 @@
+<div class="m-content">
+	<div class="row">
+		<div class="col-lg-12">
+			<!--begin::Portlet-->
+			<div class="m-portlet">
+				<div class="m-portlet__head">
+					<div class="m-portlet__head-caption">
+						<div class="m-portlet__head-title">
+							<span class="m-portlet__head-icon m--hide">
+								<i class="la la-gear"></i>
+							</span>
+							<h3 class="m-portlet__head-text">
+								Add Product
+							</h3>
+						</div>
+					</div>
+				</div>
+
+				<!--begin::Form-->
+				<form class="m-form m-form--fit m-form--label-align-right m-form--group-seperator-dashed" [formGroup]="productForm">
+					<div class="m-portlet__body">
+						<div class="form-group m-form__group row">
+							<label class="col-lg-2 col-form-label">ASIN NAME:</label>
```

Comment:
remove name from here. ASIN is not a name
`Amazon Standard Identification Number` use this for place holder.